bson/json_util.py

```diff
@@ -824,6 +831,14 @@ def _parse_canonical_maxkey(doc: Any) -> MaxKey:
     return MaxKey()
 
 
+def _encode_bson(obj: Any, json_options: JSONOptions) -> Any:
```

Comment:
Can this be removed? Or is it just an artifact from rebasing?

Reply:
Artifact from rebasing, removed.